custom.py

```diff
@@ -61,6 +61,30 @@ def dashboard():
                     db_session.commit()
                 except:
                     flash(u'Sorry, was unable to delete that worker.  Perhaps they were already deleted!', 'error')
+        elif request.form['mode']=='refresh':
+            failed_workers = []
+            workers = LegitWorker.query.all()
+            for w in workers:
+                participant_exists = Participant.query.\
+                        filter(Participant.workerid == w.amt_worker_id).count() > 0
+                if participant_exists:
+                    try:
+                        user = Participant.query.\
+                            filter(Participant.workerid == w.amt_worker_id).one()
```

Comment:
`one()` will 'fail' right here if the worker hasn't done their makeup hit yet. Is it necessary to flash this case as a failure? I suspect your try-except was intended to just catch db commit failures?